test/AuraRouterTest.php

```diff
@@ -37,6 +39,9 @@ class AuraRouterTest extends TestCase
     /** @var AuraGenerator */
     private $auraGenerator;
 
+    /** @var MiddlewareInterface|ObjectProphecy */
```

Comment:
Please remove `ObjectProphecy`, it's mocked MiddlewareInterface instance.

Reply:
Fixed and force-pushed.